src/projectkb/alice/projectkb_alice.json

```diff
@@ -0,0 +1,199 @@
+{
+	"name": "ProjectKB Alice",
+	"vendorId": "0x0159",
+	"productId": "0xA71C",
+	"lighting": {"extends":"qmk_backlight_rgblight", "keycodes": "qmk"},
+	"matrix": { "rows": 5, "cols": 16 },
+	"layouts": {
+		"labels":[
+		"Long Backspace",
+		"Long RShift"
+		],
+		"keymap": 
```

Comment:
VIA now supports rotated keys... perhaps use one of the other Alice layouts that already do rotated keys and use that as a starting point.

Reply:
Hi Wilba

I do have a version that has rotated keys. However, the rotated keys do not play well with the GUI. It is perfectly functional, but 2 visual issues arise:

1. Design tab + show matrix is not displayed properly
![image](https://user-images.githubusercontent.com/45778591/77014806-35307d00-69c7-11ea-839b-720e426b92c1.png)

2. The pink box that appears behind the key selected for mapping key inputs does not align with the key itself (will provide a screenshot later).

Reply:
OOF... I suggest commiting the KLE with rotated keys since there already are layouts with rotated keys in VIA so we have to fix these bugs (and they are pretty easy to fix I think).

Reply:
Noted, will do so when I get home.

Reply:
Screenshot of issue #2 that I mentioned earlier:
![image](https://user-images.githubusercontent.com/45778591/77055832-ac94f980-6a25-11ea-885d-5fd07987a9cd.png)
Key with 6 is selected

Reply:
Comitted the KLE with rotated keys

Reply:
Can you please check if these issues are fixed in VIA 1.2.6?

Reply:
![image](https://user-images.githubusercontent.com/45778591/77113262-9404fd80-6a7e-11ea-8cbe-9beec9300b84.png)

![image](https://user-images.githubusercontent.com/45778591/77113276-9b2c0b80-6a7e-11ea-8315-6d2039b0c6c0.png)

Works perfectly now!